activestorage/app/controllers/active_storage/representations/proxy_controller.rb

```diff
@@ -3,12 +3,10 @@
 # Proxy files through application. This avoids having a redirect and makes files easier to cache.
 class ActiveStorage::Representations::ProxyController < ActiveStorage::BaseController
   include ActiveStorage::SetBlob
-  include ActiveStorage::SetHeaders
 
   def show
     http_cache_forever public: true do
-      set_content_headers_from representation.image
-      stream representation
+      stream_from_storage representation.image
```

Comment:
Isn't this code supposed to stream the representation instead of the image? I think that is the reason why the tests are broken.

Reply:
They were broken differently before. Think we need the image for the content type headers, but the representation for the streaming.

Reply:
Did found this weird. That you can't just stream the representation directly. Maybe @georgeclaghorn knows why that is.

Reply:
All representations are supposed to produce an image, so I think this is correct?